### ISSUE TITLE
linux_nightly_coverage qt_gui_cpp test fails 

### ISSUE THREAD

**Issue by jacobperron** [ros2/build_farmer#158]
Failing for the past 6 builds:
https://ci.ros2.org/view/nightly/job/nightly_linux_coverage/347/

**qt_gui_cpp.test.test_imports.test_import_libqt_gui_cpp_sip**
```
ImportError: /home/rosbuild/ci_scripts/ws/install/qt_gui_cpp/lib/python3.6/site-packages/qt_gui_cpp/libqt_gui_cpp_sip.so: undefined symbol: __gcov_merge_add
```

Stack Trace:
```
def test_import_libqt_gui_cpp_sip():
>       from qt_gui_cpp import libqt_gui_cpp_sip
E       ImportError: /home/rosbuild/ci_scripts/ws/install/qt_gui_cpp/lib/python3.6/site-packages/qt_gui_cpp/libqt_gui_cpp_sip.so: undefined symbol: __gcov_merge_add

test/test_imports.py:17: ImportError
```

**Comment by jacobperron** [comment]
This failure only occurs with the coverage cmake flags. To reproduce locally:

```
colcon build --symlink-install --cmake-args -DCMAKE_CXX_FLAGS="-fprofile-arcs -ftest-coverage" --packages-up-to qt_gui_cpp
colcon test --packages-select qt_gui_cpp
```

**Comment by dirk-thomas** [comment]
Addressed during #168.
